src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -92,60 +98,94 @@ const PIACollectionUseAndDisclosure = () => {
+              <a
+                href={Messages.CollectionNotice.DrafterInput.Link.en}
+                rel="noreferrer external"
+                target="_blank"
+              >
+                {Messages.CollectionNotice.DrafterInput.Title.LinkText.en}
+                <FontAwesomeIcon
+                  className="helper-text__link-icon"
+                  icon={faUpRightFromSquare}
+                />
+              </a>
+              {Messages.CollectionNotice.DrafterInput.Title.PartThree.en}
+            </label>
+          ) : (
+            <h4>
+              {Messages.CollectionNotice.DrafterInput.Title.PartOne.en}
               {Messages.CollectionNotice.DrafterInput.Title.LinkText.en}
-              <FontAwesomeIcon
-                className="helper-text__link-icon"
-                icon={faUpRightFromSquare}
-              />
-            </a>
-            {Messages.CollectionNotice.DrafterInput.Title.PartThree.en}
-          </label>
-
-          <div className="section__question-hint">
-            {Messages.CollectionNotice.DrafterInput.Description.en}
-          </div>
+              {Messages.CollectionNotice.DrafterInput.Title.PartThree.en}
+            </h4>
+          )}
+          {!isReadOnly && (
+            <div className="section__question-hint">
+              {Messages.CollectionNotice.DrafterInput.Description.en}
+            </div>
+          )}
           <div className="richText" id="drafterDisclosure">
-            <MDEditor
-              id="collectionNoticeDrafter"
-              preview="edit"
-              value={
-                collectionUseAndDisclosureForm?.collectionNotice?.drafterInput
-              }
-              defaultTabEnable={true}
-              onChange={(value) => {
-                stateChangeHandler(value, 'collectionNotice.drafterInput');
-              }}
-            />
+            {isReadOnly ? (
+              <MDEditor.Markdown
+                source={
+                  collectionUseAndDisclosureForm?.collectionNotice?.drafterInput
+                }
+              />
+            ) : (
+              <MDEditor
+                id="collectionNoticeDrafter"
+                preview="edit"
+                value={
+                  collectionUseAndDisclosureForm?.collectionNotice?.drafterInput
+                }
+                defaultTabEnable={true}
+                onChange={(value) => {
+                  stateChangeHandler(value, 'collectionNotice.drafterInput');
+                }}
+              />
+            )}
           </div>
         </div>
         <div className="form-group">
-          <label htmlFor="collectionNoticeMPO" className="pt-5">
-            {Messages.CollectionNotice.MpoInput.Title.en}
-          </label>
+          {!isReadOnly ? (
+            <label htmlFor="collectionNoticeMPO" className="pt-5">
+              {Messages.CollectionNotice.MpoInput.Title.en}
+            </label>
+          ) : (
+            <h4 className="pt-5">
```

Comment:
Personally, I might ask UI/UX to keep the title tags the same for both view and edit.

Reply:
Only `h1` and `h2` tags are a heavier font weight, I think we should maintain that design language as it creates a consistent and visually pleasing hierarchy.

That said, I do need to clarify, with UI/UX, what to do with empty fields when in view mode.

Reply:
I have followed up with Jordan. She will reach out to the POs and return a verdict on how to handle unanswered questions in view mode.